client/src/components/frontend/Resource/Detail.js

```diff
@@ -10,12 +10,14 @@ export default class ResourceDetail extends Component {
   static propTypes = {
     projectId: PropTypes.string,
     projectUrl: PropTypes.string,
+    resourceUrl: PropTypes.string,
```

Comment:
It's OK to not have this resourceUrl be required, but if it's not, then we shouldn't render the ShareBar if it's absent or blank. Or, alternately, we can make it required and always render the ShareBar.